### PR TITLE
add browser detection

### PR DESCRIPTION
**Pull request by pein0119**
bowser - a browser detector

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ A collection of awesome browser-side  JavaScript libraries, resources and shiny
     * [Voice Command](#voice-command)
     * [API](#api)
     * [Vision Detection](#vision-detection)
+	* [Browser Detection](#browser-detection)
```
Comment (huei90) [inline README.md]: format incorrect?

**Comment by huei90** [comment]
@pein0119 Nice project. But could you mind update the format?

**Comment by huei90** [comment]
Open a new PR and I will merge it. :+1:

**Comment by pein0119** [comment]
ok, I will do it later

**Comment by pein0119** [comment]
https://github.com/sorrycc/awesome-javascript/pull/161
done

**Pull request closed without merging**